piezo/TDC001.py

```diff
@@ -0,0 +1,134 @@
+            timeout = 0
+            while not(self.device.IsSettingsInitialized()) and (timeout <= 10):
+                self.device.WaitForSettingsInitialized(500)
+                timeout += 1
+            self.device.StartPolling(250)
+            time.sleep(0.5)
+            self.device.EnableDevice()
+            self.device_info = self.device.GetDeviceInfo()
+            self.short_name = self.device_info.Name
+            print("Success ! Connected to TCube motor:" +
+                  f" {self.device_info.Description}" + 
+                  f", S/N: {self.device_info.SerialNumber}"
+                  )
+        except Exception:
+            print("ERROR : Could not connect to the device")
+            print(traceback.format_exc())
+
+    def __is_command_complete(self, taskID: int):
+        """Private method to handle completion of tasks
+
+        :param int taskID: Task whose status is being querried
+        :return: None
+        :rtype: Nonetype
+
+        """
+        if self.__taskID > 0 and self.__taskID == taskID:
+            self.__taskComplete = True
+
+    def home(self, timeout: float = 60e3) -> bool:
+        """Homes the device to its center position. Might take some time.
+
+        :param float timeout: Timeout of the movement.
+        :return bool isHomed: If the device is homed
+
+        """
+        print(f"{self.short_name} | Homing device...")
+        try:
+            self.device.Home(int(timeout))
+            print(f"{self.short_name} | Device homed !")
+        except Exception:
+            print(f"{self.short_name} | ERROR : Could not home the device")
+            print(traceback.format_exc())
+        # self.__taskComplete = false
+        # Action = getattr(System, "Action`1")
+        # checkcomplete = Action[UInt64](self.__is_command_complete)
+        # self.__taskID = self.device.Home(checkcomplete)
+        # t0 = time.time()
+        # waittime = 0
+        # while not(self.__taskComplete) and waittime < timeout:
+        #     time.sleep(500e-3)
+        #     status = self.device.Status
+        #     sys.stdout.write(f"\rHoming device ... Position : {status.Position}")
+        #     waittime = time.time()-t0
+
+    def move_to(self, pos: float, timeout: float = 60e3):
+        """Simple move, returns final position in device coordinates
+
+        :param float pos: Position
+        :param float timeout: Timeout in ms to do the move
+        :return: None
+        :rtype: float
+
+        """
+        print(f"{self.short_name} | Moving to {pos}°")
+        self.device.MoveTo(Decimal(pos), int(timeout))
+        print(f"{self.short_name} | Device position: { self.device.Position }°")
+        return float(str(self.device.Position))
+
+    def get_position(self):
+        """Returns the actual position
+
+        :return: Position
+        :rtype: float
+
+        """
+        print(f"{self.short_name} | Device position: { self.device.Position }°")
```

Comment:
Minor: I would remove the degree symbol because this DC motor can be used for translation stages.